lib/liftoff/xcodeproj_helper.rb

```diff
@@ -53,18 +53,20 @@ def add_script_phases(scripts)
       end
     end
 
-    def perform_extra_config(extra_config)
-      if extra_config
-        extra_config.each do |name, settings|
-          if name.downcase == "all"
-            object = target
-          else
-            object = target.build_settings(name)
-          end
+    def perform_extra_config(app_config, test_config)
+      [app_config, test_config].each do |config|
+        if config
```

Comment:
You could do `.compact.each` to remove this `if` statement. `.compact` removes `nil`s.